0000-template.md

```diff
@@ -1,23 +1,30 @@
 ---
-Stage: Accepted
-Start Date: 
-Release Date: Unreleased
-Release Versions:
-  ember-source: vX.Y.Z
-  ember-data: vX.Y.Z
-Relevant Team(s): 
-RFC PR: 
+stage: accepted
+start-date: 
+release-date: 
+release-versions:
+teams: # delete teams that aren't relevant
+  - cli
+  - data
+  - framework
+  - learning
+  - steering
+  - typescript
+prs:
+  accepted: # update this to the PR that you propose your RFC in
+project-link:
 ---
 
 <!--- 
 Directions for above: 
 
-Stage: Leave as is
-Start Date: Fill in with today's date, YYYY-MM-DD
-Release Date: Leave as is
-Release Versions: Leave as is
-Relevant Team(s): Fill this in with the [team(s)](README.md#relevant-teams) to which this RFC applies
-RFC PR: Fill this in with the URL for the Proposal RFC PR
+stage: Leave as is
+start-date: Fill in with today's date, 2032-12-01T00:00:00.000Z
```

Comment:
The linter doesn't require the time, maybe we shouldn't?

Reply:
We discussed this a bit in the Discord chat, essentially we want to mark those dates as UTC to prevent a whole host of issues down the line. I was under the impression that we concluded this wasn't a blocker 🤔 

If the issue is that the linter doesn't require this format are you asking me to add this to the linter?

Reply:
If you want it to always have the time part, you should definitely add it to the linter. I'm sure most authors will write `YYYY-MM-DD` no matter what is in instructions, otherwise.

Reply:
Out of curiosity, what's a concrete example of a problem that would arise in this app, and which we would avoid by using this format?